app/src/components/pages/ScheduleTimetable.tsx

```diff
@@ -1,26 +1,54 @@
 import React from 'react';
-import Scheduler from 'devextreme-react/scheduler';
+import { Scheduler, Editing } from 'devextreme-react/scheduler';
 import 'devextreme/dist/css/dx.light.css';
 import Appointment from 'components/organisms/Appointment';
 import classData from 'data/clean.json';
-import { parseCalendarJSON } from 'utils/utils';
+import { parseCalendarJSON_Course, parseCalendarJSON_Teacher } from 'utils/utils';
+import { Resource } from 'devextreme-react/scheduler';
 
 //The current date will be +1 month in the UI, ex: 2021/Dec/10 -> 2022/Jan/10
 const currentDate = new Date(2021, 12, 10);
 
 function ScheduleTimetable() {
-  const calendarData = parseCalendarJSON(JSON.parse(JSON.stringify(classData)));
+  const calendarData_Course = parseCalendarJSON_Course(JSON.parse(JSON.stringify(classData)));

```

Comment:
You can get rid of the `_` here, we want to consistently use camelCase in our repo